packages/web/docs/src/components/get-in-touch-button.tsx

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import { CallToAction } from '@theguild/components';
+
+export const GetInTouchButton = () => {
```

Comment:
Is it used in more than 1 place?

Reply:
https://github.com/graphql-hive/console/pull/6089/files/6ba874b555ff81995bc393583e8c6ad9394bb6f7#r1890528043

Same as here

```tsx
<ContactButton variant="primary-inverted">Get in touch</ContactButton>
```